ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -101,7 +101,10 @@ export default class ExperimentalTab extends PureComponent {
               >
                 {t('selectProvider')}
               </Text>
-              <div className="settings-page__content-item-col settings-page__content-item-col-open-sea">
+              <div
+                data-testid="transaction-security-check-blockaid"
```

Comment:
how were the data-testid values chosen? I wonder if it would help prepending the filename to avoid naming collisions

Reply:
Do you think this is better? https://github.com/MetaMask/metamask-extension/pull/20454/commits/9767c468d9747750dec30c5825691a89b8397213#diff-a342e4508644fdc378f3acf43755a2fa484916f3eac050df4e2059927c128709R92

Reply:
maybe `settings-toggle-security-alert-blockaid` to include the filename. Usually I recommend more of the filepath to avoid collisions, but I think each setting can be unique enough to prepend with `setting` alone

Reply:
Updated here. https://github.com/MetaMask/metamask-extension/pull/20454/commits/f4a7997dd849355db8cf9ebe49eefd2bc7cc046f